data/eval-machines.nix

```diff
@@ -6,7 +6,12 @@ let
   nwPkgs = network.network.pkgs or { };
   lib = network.network.lib or nwPkgs.lib or (import <nixpkgs/lib>);
   evalConfig =
-    network.network.evalConfig or ((nwPkgs.path or <nixpkgs>) + "/nixos/lib/eval-config.nix");
+    let
+      evalConfig' =
+        network.network.evalConfig or ((nwPkgs.path or <nixpkgs>) + "/nixos/lib/eval-config.nix");
+    in
+    if lib.isFunction evalConfig' then evalConfig' else import evalConfig';
```

Comment:
Is it ever safe to import this here when we unconditionally do the same in `nodes` and `uncheckedNodes`?

Reply:
no one knows

Reply:
I can't immediate think of a way in which this makes sense. This change breaks morph (`nix run -- build examples/simple.nix`)

I suggest either dropping the change or figuring out what its real purpose was and fixing its implementation. :)

Reply:
I've dropped the change for now